install/preview/entrypoint.sh

```diff
@@ -136,6 +136,18 @@ yq eval-all -i 'del(.spec.template.spec.initContainers[0])' /var/lib/rancher/k3s
 for f in /var/lib/rancher/k3s/server/manifests/gitpod/*.yaml; do (cat "$f"; echo) >> /var/lib/rancher/k3s/server/manifests/gitpod.yaml; done
 rm -rf /var/lib/rancher/k3s/server/manifests/gitpod
 
+# waits for gitpod pods to be ready, and manually runs the `gitpod-telemetry` cronjob
+run_telemetry(){
+  # wait for the k3s cluster to be ready and Gitpod workloads are added
+  sleep 100
+  # indefinitely wait for Gitpod pods to be ready
+  kubectl wait --timeout=7m --for=condition=ready pod -l app=gitpod,component!=migrations
```

Comment:
Are 7 minutes long enough? What if it takes longer to pull the images due to bad internet connectivity?

Reply:
Makes sense. Updated it to be `-1s` which means a week, as users are not expecting feedback from this command, and we can wait more time. wdyt? 🤔